gallery/src/Application.tsx

```diff
@@ -4,92 +4,90 @@
 
 function Application() {
   return (
-    <Router>
-      <PageHeader />
-      <div className="nx-page-content">
-        <Switch>
-          <Route exact path="/PageLevelAlertExample">
-            <NxLoadWrapperPageLevelExample/>
-          </Route>
-          <Route>
-            <aside className="nx-page-sidebar" id="gallery-sidebar">
-              <GalleryNav />
-            </aside>
-            <Switch>
-              <Route path="/pages/:pageName" component={Page} />
-              <Route exact path="/" component={Page} />
-
-              {/* Special cases, these examples need their own page separate from their documentation */}
-              <Route exact path="/NxViewportSizedExample">
-                <SectionScrollingWrapper>
-                  <NxViewportSizedExample />
-                </SectionScrollingWrapper>
-              </Route>
-              <Route exact path="/NxViewportSizedExpandingExample">
-                <SectionScrollingWrapper>
-                  <NxViewportSizedExpandingExample />
-                </SectionScrollingWrapper>
-              </Route>
-              <Redirect to="/" />
-            </Switch>
-          </Route>
-        </Switch>
+    <div id="temporary-navigation-sidebar-example" className="nx-page-content nx-page-content--full-width">
+      <NxStatefulNavigationSidebar isDefaultOpen={true}
+                                   toggleOpenIcon={faBiohazard}
+                                   toggleCloseIcon={faCrow}
+                                   logoText="nexus lifecycle"
+                                   logoLink="/home">
+        <NxNavigationSidebarLinks>
+          <a href="/alpha" className="selected">
+            <NxFontAwesomeIcon icon={faAtom} fixedWidth />
+            <span>Link Name Alpha</span>
+          </a>
+          <a href="/beta">
+            <NxFontAwesomeIcon icon={faBiohazard} fixedWidth />
+            <span>Link Name Beta</span>
+          </a>
+          <a href="/gamma">
+            <NxFontAwesomeIcon icon={faPlaceOfWorship} fixedWidth />
+            <span>Link Name Gamma</span>
+          </a>
+          <a href="/delta">
+            <NxFontAwesomeIcon icon={faCrow} fixedWidth />
+            <span>Link Name delta</span>
+          </a>
+          <a href="/epsilon">
+            <NxFontAwesomeIcon icon={faBolt} fixedWidth />
+            <span>Link Name Epsilon</span>
+          </a>
+          <a href="/Zeta">
+            <NxFontAwesomeIcon icon={faAtom} fixedWidth />
+            <span>Link Name Zeta</span>
+          </a>
+          <a href="/Etha">
+            <NxFontAwesomeIcon icon={faPlaceOfWorship} fixedWidth />
+            <span>Link Name Etha</span>
+          </a>
+          <a href="/Theta">
+            <NxFontAwesomeIcon icon={faCrow} fixedWidth />
+            <span>Link Name Theta</span>
+          </a>
+        </NxNavigationSidebarLinks>
+        <NxNavigationSidebarContent className="navigation-sidebar-custom-content-1">
```

Comment:
This class name (and the one on the footer) makes it sound like these sections are not standardized but would be custom to a particular app.  Is that the intent?

Reply:
> This class name (and the one on the footer) makes it sound like these sections are not standardized but would be custom to a particular app. Is that the intent?

Yes. This code is just _an_ example of one possible implementation out of many.

Reply:
> Yes. This code is just an example of one possible implementation out of many.

So this way of displaying the help, version number, etc, is expected to be IQ specific? In that case given that this is just placeholder content here in RSC you could probably simplify it quite a bit here.

Reply:
> > Yes. This code is just an example of one possible implementation out of many.
> 
> So this way of displaying the help, version number, etc, is expected to be IQ specific? In that case given that this is just placeholder content here in RSC you could probably simplify it quite a bit here.

See my note in the PR description about _all_ modified code inside `gallery` being throwaway code — this is code to temporary show how the components can be consumed, it is in no way code to be merged.

Reply:
> See my note in the PR description about all modified code inside gallery being throwaway code — this is code to temporary show how the components can be consumed, it is in no way code to be merged.

But I remain skeptical that this is true:

> So this way of displaying the help, version number, etc, is expected to be IQ specific?

If it isn't, then that aspect should not be throwaway code after all.

Reply:
> So this way of displaying the help, version number, etc, is expected to be IQ specific?
It is. Therefore is not something to be implemented _here_.